src/script.js

```diff
@@ -441,6 +461,7 @@ module.exports = {
   isMultisigOutput: isMultisigOutput,
   isNullDataOutput: isNullDataOutput,
 
+  parseMultisigScript: parseMultisigScript,
```

Comment:
If we are, we probably shouldn't use `m` and `n` as names because that convention came from multisig? least it might be less obvious what they're for if used for p2pkh.